setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     pyyaml>=3.10
     jsonschema>=3.0.2,<4
     numpy>=1.10
+    importlib_resources>=3;python_version<"3.9"
```

Comment:
What is `;python_version<"3.9"` doing here?  I believe `importlib_resources` is only needed for `python<=3.6`

Reply:
I'm using `importlib.resources.files`, which is implemented in the shim library but only becomes available in 3.9.

Reply:
Ah, ok!  Interesting.  Didn't know about that feature.